### DISCUSSION TITLE
How to get data of different model in MST?

### DISCUSSION THREAD

**Discussion by mrshahzeb7** [infinitered/ignite#2503] [Q&A] [answered]
This is the parent store 

  .model("PropertyStore")
  .props({
    propertyListModel: types.optional(PropertyListModel, {}),
    propertyDetailsModel: types.optional(PropertyDetailsModel, {}),
  })
  
  I have a list in "PropertyListModel" that I want to get inside "PropertyDetailsModel" 
 How can I do that?

This is the propertyDetailsModel.ts

export const PropertyDetailsModel = types
  .model("propertyDetailsModel")
  .props({
    selectedPropertyDetails: types.frozen(),
  })
  .actions(withSetPropAction)
  .actions((self) => ({
    async getPropertyDetailsById(id?: number) {
      
      //Here I want to get that List. 
      // self._root.propertyListModel
    
    },
  }))

**Comment by frankcalise** [comment, marked as answer]
hey @mrshahzeb7 , you can do this by using the `getRootStore` helper here: https://github.com/infinitered/ignite/blob/master/boilerplate/app/models/RootStore.ts

So, for example if your root store looked something like this:
```
export const RootStoreModel = types.model("RootStore").props({
  propertyListStore: types.optional(PropertyListModel, {}),
  propertyDetailsStore: types.optional(PropertyDetailsModel, {}),
})
```

You could then call on a property from the list store in detail store with:

`getRootStore(_self).propertyListStore.myProp`

**Reply by mrshahzeb7** [reply]
Thankyou got it 👍 

Kindly update the path 
"https://github.com/infinitered/ignite/blob/master/boilerplate/app/models/helpers/getRootStore.ts"
